### ISSUE TITLE
rmh-simulation of ppm fit in an alternative window (1.41-1)

### ISSUE THREAD

**Issue by antiphon** [spatstat/spatstat#9]
spatstat version 1.41-1
R version 3.1.2

Trying to simulate a fitted model in an alternative window gives me an error.

Example:

```
library(spatstat)
x <- rpoispp(100)
f <- ppm(x~1, i=StraussHard(0.1))
Wnew <- as.owin(c(0,2,0,2))
simulate(f, w=Wnew)
Error in runifpoint(npts.free, w.sim, ...) : 
  argument 3 matches multiple formal arguments

rmh(f, w=Wnew)
Error in rmh.default(model = list(cif = "straush", par = list(gamma = 0.96592610760324,  : 
  Expanded simulation window does not contain model window

```

**Comment by baddstats** [comment]
Just confirming that these problems also occur in the current version of spatstat (1.42-2) and R (3.2.1). 
I will investigate.

**Comment by baddstats** [comment]
These bugs have now been fixed (in version 1.42-2.002)
Thank you!
